blob-storage/instance.tf

```diff
@@ -3,7 +3,7 @@ resource "azurerm_virtual_machine" "demo-instance" {
   name                  = "${var.prefix}-vm"
   location              = var.location
   resource_group_name   = azurerm_resource_group.demo.name
-  network_interface_ids = ["${azurerm_network_interface.demo-instance.id}"]
+  network_interface_ids = ["azurerm_network_interface.demo-instance.id"]
```

Comment:
here and in the next ones, the quotes are still there, so those should be removed

Reply:
Thanks, I will update the PR

Reply:
@wardviaene I have removed the quotes.